packages/block-library/src/table/style.scss

```diff
@@ -4,15 +4,21 @@
 	$subtle-pale-blue: #e7f5fe;
 	$subtle-pale-pink: #fcf0ef;
 
+	overflow-x: auto;
+
+	table {
+		width: 100%;
+	}
+
 	// Fixed layout toggle
-	&.has-fixed-layout {
+	.has-fixed-layout {
```

Comment:
Seeing some inconsistencies between the editor and the post when it comes to the fixed layout. The editor seems to wrap long text, but when viewing the post text isn't wrapped.

Reply:
Fixed 😅